### PR TITLE
Spreadsheet: Add missing examples

### PR DESCRIPTION
**Pull request by monroeclinton**
This patches adds examples for the `column`, `range`, `row`, and `now` functions for the Spreadsheet program.

### CODE REVIEW

File: Base/res/js/Spreadsheet/runtime.js [hunk 1]
```diff
@@ -403,7 +407,9 @@ randRange.__documentation = JSON.stringify({
     argc: 2,
     argnames: ["start", "end"],
     doc: "Returns a random number in the range (`start`, `end`)",
-    examples: {},
+    examples: {
+        'range("A1", "C4")': "Returns a random number within A1:C4"
```
Comment (alimpfard) [inline Base/res/js/Spreadsheet/runtime.js:411]: This is not how it works, `range` gives you...well, a range
randRange doesn't take cell names, it takes numbers (it's basically `Math.random`)

File: Base/res/js/Spreadsheet/runtime.js [hunk 2]
```diff
@@ -385,7 +385,11 @@ now.__documentation = JSON.stringify({
     argc: 0,
     argnames: [],
     doc: "Returns a Date instance for the current moment",
-    examples: {},
+    examples: {
+        "now()": "Returns an instance of a Date object",
```
Comment (alimpfard) [inline Base/res/js/Spreadsheet/runtime.js:389]: Isn't this example a bit redundant? It's exactly the description plus the function

File: Base/res/js/Spreadsheet/runtime.js (hunk 1 above)
Comment (monroeclinton) [reply Base/res/js/Spreadsheet/runtime.js]: You're correct, I'm not sure what I was looking at last night

File: Base/res/js/Spreadsheet/runtime.js (hunk 2 above)
Comment (monroeclinton) [reply Base/res/js/Spreadsheet/runtime.js]: It is, I removed it. I wasn't sure if the base function should always be included in examples

**Review by alimpfard — COMMENTED** [review]
Please run `prettier` on the `runtime.js` file (that's what the CI is sad about).

**Pull request merged**